index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <h2>Welcome</h2>
+    <nav>
+      <span id="humburger"><i class="fa humburger fa-bars"></i></span>
+    </nav>
```

Comment:
- [ ] Great job for adding the hamburger to your design, it looks really nice. so I have noticed that the hamburger is actually inside a `span` tag. Can you please try to put inside a `button` tag instead of  `span `  tag just to follow the semantic Html and as long as It plays the role of the button.